internal/handler/wallet/postcondition_test.go

```diff
@@ -0,0 +1,76 @@
+package wallet
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/massalabs/thyra-plugin-massa-wallet/api/server/models"
+)
+
+// Clean up test data by listing all created wallets with tests and deleting them.
+func cleanupTestData(t *testing.T) {
```

Comment:
Why do you need a testing.T parameter ?

Reply:
I guess it will be a lot easier to simply remove (delete) the file on the file system ....